test-data/unit/fixtures/typing-full.pyi

```diff
@@ -10,7 +10,7 @@ from abc import abstractmethod, ABCMeta
 
 class GenericMeta(type): pass
 
-def cast(t, o): ...
+def cast(t: Any, o: Any) -> Any: ...
```

Comment:
what's this? should it use `object` instead

Reply:
I only typed it because a test that was using it complained. It used to be `Any` do it doesn't even matter I think.